diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -178,6 +179,10 @@ namespace diff_drive_controller{
     controller_nh.param("enable_odom_tf", enable_odom_tf_, enable_odom_tf_);
     ROS_INFO_STREAM_NAMED(name_, "Publishing to tf is " << (enable_odom_tf_?"enabled":"disabled"));
 
+    controller_nh.param("estimate_velocity_from_position", estimate_velocity_from_position_, estimate_velocity_from_position_);
+    ROS_INFO_STREAM_NAMED(name_, "Estimating joint velocity from joint position is  " <<
+        (enable_odom_tf_?"enabled":"disabled"));
```

Comment:
This is a bit redundant, I guess you meant `estimate_velocity_from_position_` here :)